<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="fisheye_camera">
    <static>true</static>
    <link name="link">
      <sensor name="camera" type="camera">
        <camera>
          <horizontal_fov>1.5708</horizontal_fov>
          <image>
            <width>1280</width>
            <height>720</height>
            <format>R8G8B8</format>
          </image>
          <clip>
            <near>0.1</near>
            <far>100</far>
          </clip>
          <distortion>
            <k1>-0.2</k1>
            <k2>0.0</k2>
            <k3>0.0</k3>
            <p1>0.0</p1>
            <p2>0.0</p2>
            <center>0.5, 0.5</center>
          </distortion>
        </camera>
        <always_on>true</always_on>
        <update_rate>30.0</update_rate>
        <visualize>true</visualize>
      </sensor>
    </link>
  </model>
</sdf>